dev/TeachingTip/TeachingTip.idl

```diff
@@ -84,21 +75,19 @@ unsealed runtimeclass TeachingTip : Windows.UI.Xaml.Controls.ContentControl
     TeachingTip();
 
     String Title;
-    String Subtext;
+    String Subtitle;
 
     [MUX_DEFAULT_VALUE("false")]
     Boolean IsOpen;
 
     Windows.UI.Xaml.FrameworkElement Target;
 
-    String ActionButtonText;
+    Object ActionButtonText;
```

Comment:
In the API review I requested that this and CloseButtonText be called CloseButton**Content** since they are object-typed. Can you make those changes here too?